src/key.ml

```diff
@@ -78,32 +78,32 @@ type t =
   | Z
 
   (* Alphabet keys (lowercase) *)
```

Comment:
This comment doesn't match the codes now (and does the one above the ones above?)